python/src/spark_rapids_ml/core.py

```diff
@@ -614,7 +617,9 @@ def _cuml_transform(cuml_obj: CumlT, df: Union[pd.DataFrame, np.ndarray]) ->pd.D
                 ...
             ...
 
-            return _construct_cuml_object, _cuml_transform
+            array_order = "F"
```

Comment:
array_order seems like a cuml param. Thinking it may be better to move it to the _CumlParams class.

Reply:
It is not passed into the cuml object so not really consistent with that class as it stands.

Reply:
Created separate array_order methods for fit and transform (in case they are different) with most common case "F" as the default return value.